sdk/examples/command_center.ts

```diff
@@ -0,0 +1,207 @@
+        url,
+    });
+
+    const iotaNamesClient = new IotaNamesClient({
+        client: iotaClient,
+        network: 'devnet',
+    });
+
+    console.log('[Domain]:', domain);
+    console.log('[Secret key]:', keypair.getSecretKey());
+    console.log('[Address]:', address);
+
+    switch (cmd) {
+        case 'bid':
+            {
+                const bidAmount = 1n * NANOS_PER_IOTA;
+
+                const auctionHouse = await getAuctionHouse(iotaNamesClient, graphql!);
+
+                const tx = new Transaction();
+                const iotaNamesTx = new IotaNamesTransaction(iotaNamesClient, tx);
+
+                const bid = iotaNamesTx.transaction.splitCoins(iotaNamesTx.transaction.gas, [
+                    bidAmount,
+                ]);
+
+                await startAuctionAndPlaceBid(iotaNamesTx, auctionHouse, domain, bid);
+
+                iotaNamesTx.transaction.setSender(address);
+
+                const transaction = await iotaNamesTx.transaction.build({
+                    client: iotaClient,
+                });
+
+                const executedTransactionResponse = await iotaClient.signAndExecuteTransaction({
+                    transaction,
+                    signer: keypair,
+                });
+
+                console.log('[Transaction digest]:', executedTransactionResponse.digest);
+
+                await iotaClient.waitForTransaction({
+                    digest: executedTransactionResponse.digest,
+                });
+
+                const transactionBlockResponse = await iotaClient.getTransactionBlock({
+                    digest: executedTransactionResponse.digest,
+                    options: {
+                        showEffects: true,
+                    },
+                });
+
+                console.log(transactionBlockResponse);
+            }
+            break;
+
+        case 'check-own': {
+            console.log(
+                '[Domain Record (null if available)]: ',
+                await iotaNamesClient.getNameRecord(domain),
+            );
+
+            break;
+        }
+
+        case 'create': {
+            await requestIotaFromFaucetV0({
+                host: faucet!,
+                recipient: address,
+            });
+
+            console.log('[Faucet]: Received funds');
+
+            console.log(
+                '[Domain Record (null if available)]: ',
+                await iotaNamesClient.getNameRecord(domain),
+            );
+
+            const tx = new Transaction();
+            const iotaNamesTx = new IotaNamesTransaction(iotaNamesClient, tx);
+            const [coin] = iotaNamesTx.transaction.splitCoins(tx.gas, [100_000_000]);
+
+            const nft = iotaNamesTx.register({
+                domain,
+                years: 1,
+                coin,
+            });
+
+            iotaNamesTx.transaction.transferObjects([nft], address);
+
+            iotaNamesTx.transaction.transferObjects([coin], address);
```

Comment:
Can't you just do this?
```suggestion
            iotaNamesTx.transaction.transferObjects([coin, nft], address);
```

Reply:
:trollface: Yes